PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -1,11 +1,13 @@
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.NanoAOD.common_cff import *
-from PhysicsTools.NanoAOD.nano_eras_cff import run3_nanoAOD_124
+from PhysicsTools.NanoAOD.nano_eras_cff import *
```

Comment:
why ?

Reply:
There is not any particular reason. I think I changed it for some test and then not reverted. It can be changed back, however I do not see problem with importing all eras. Should I revert it?

Reply:
I prefer the import of only the things required